kubernetes/ingress/zooniverse.org.yaml

```
@@ -300,6 +300,13 @@ spec:
           serviceName: subject-assistant-proxy
           servicePort: 80
         path: /(.*)
+  - host: subject-set-search-api.zooniverse.org
+    http:
+      paths:
+      - backend:
+          serviceName: subject-set-search-api-production-app
+          servicePort: 80
+        path: /(.*)
```

Not blocking: we don't do any regex path rewriting and i'd like to move away from using it on our ingresses by default (a vestige of a how-to example)
```suggestion
        path: /
```